Plugin/DotpayDirectPlugin.php

```diff
@@ -497,4 +499,21 @@ protected function getReturnUrl(ExtendedDataInterface $data)
 
         throw new \RuntimeException('You must configure a return url.');
     }
+
+    protected function getCallBackUrl(PaymentInstructionInterface $instruction, $route, $baseUrl): string
```

Comment:
Why is this method protected ?
```suggestion
    private function getCallBackUrl(PaymentInstructionInterface $instruction, $route, $baseUrl): string
```